### PR TITLE
Adding in error handling for missing homebrew packages

### PR DESCRIPTION
**Pull request by MSLaguana**

### CODE REVIEW

File: src/taco-remote-lib/ios/ios.ts [hunk 1]
```diff
@@ -218,6 +218,15 @@ class IOSAgent implements ITargetPlatform {
                 res.status(200).json(buildInfo.localize(req, resources));
             }
         });
+
+        ideviceinstaller.on("error", function (err: any): void {
+            res.status(500);
+            if (err.code === "ENOENT") {
+                res.send(resources.getStringForLanguage(req, "IdeviceinstallerNotFound"));
```
Comment (nisheetjain) [inline src/taco-remote-lib/ios/ios.ts]: Can you fix this casing as well? looks like something I regressed during camelCasing conversions

Comment (MSLaguana) [inline src/taco-remote-lib/ios/ios.ts]: Which casing are you referring to? The string ID here is new, and if you prefer a different casing then I can change it.

Comment (nisheetjain) [inline src/taco-remote-lib/ios/ios.ts]: this should be IDeviceXXX right?

Comment (MSLaguana) [inline src/taco-remote-lib/ios/ios.ts]: The name of the tool is "ideviceinstaller", so I went with "IdeviceinstallerNotFound", but it's easy enough to change. This isn't something that you broke earlier on, don't worry about that :)

**Comment by nisheetjain** [comment]
:+1:  looks good otherwise

**Pull request merged**